eng/common/scripts/ChangeLog-Operations.ps1

```diff
@@ -40,19 +40,32 @@ function Get-ChangeLogEntriesFromContent {
   $changeLogEntries = [Ordered]@{}
   try {
     # walk the document, finding where the version specifiers are and creating lists
-    $changeLogEntry = $null
     foreach ($line in $changeLogContent) {
       if ($line -match $RELEASE_TITLE_REGEX) {
         $changeLogEntry = [pscustomobject]@{ 
           ReleaseVersion = $matches["version"]
           ReleaseStatus  =  $matches["releaseStatus"]
           ReleaseTitle   = "## {0} {1}" -f $matches["version"], $matches["releaseStatus"]
           ReleaseContent = @()
+          Sections = @{}
         }
         $changeLogEntries[$changeLogEntry.ReleaseVersion] = $changeLogEntry
       }
       else {
         if ($changeLogEntry) {
+          if ($line.Trim() -match "^###\s(?<sectionName>.*)")
+          {
+            $sectionName = $matches["sectionName"].Trim()
+            $changeLogEntry.Sections[$sectionName] = @()
```

Comment:
If there are multiple sections with the same name this will essentially throw out the old section. That might not be a real issue but something to keep in mind.